x-pack/plugins/fleet/server/services/preconfiguration.ts

```diff
@@ -45,7 +45,7 @@ export async function ensurePreconfiguredPackagesAndPolicies(
     // If there are multiple packages with duplicate versions, separate them with semicolons, e.g
     // package-a:1.0.0, package-a:2.0.0; package-b:1.0.0, package-b:2.0.0
     const duplicateList = duplicatePackages
-      .map(([, versions]) => versions.map((v) => `${v.name}:${v.version}`).join(', '))
+      .map(([, versions]) => versions.map((v) => `${v.name}-${v.version}`).join(', '))
```

Comment:
Can use pkgToPkgKey https://github.com/elastic/kibana/blob/f1d662932133c49f3c2efe334b60201eeaa20a7d/x-pack/plugins/fleet/server/services/epm/registry/index.ts#L74 to ensure the correct value and avoid the duplication